### PR TITLE
Bounties tracking

### PR DESCRIPTION
**Pull request by xcorail**
Automated creation of an internal issue that replicates a bounty submission
The code is not perfect: some tests are missing, the naming is not ideal and there is a bit of duplication, but I think it's good enough to stop the spreadsheet and start with the new tracking board. 

**NOTE**: There is a workflow called `workflow-existing` that seems duplicated. Ignore it, it's just a workflow that I will use to migrate the existing issues, and it will disappear just after, so don't worry about the duplication in there   

The body of the created issue is exactly the same as the original one, prefixed with 
image

The title is the same as the original one, prefixed by `[BOUNTY - All For One]` (or other bounty type). This is bc the bot has only **Read** access to the internal repo, therefore it cannot add labels to the issue. 

At issue creation, these comments (3rd one obviously depending whether it's actually the first submission) are created on the internal issue: 

Screenshot 2020-05-21 at 22 10 41

And this one is created on the external issue:
image

### CODE REVIEW

File: .github/actions/replicate/replicate.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+export type Issue = {title: string, body: string, labels: string[]}
+export const BOUNTY_LABELS: string[] = ['All For One', 'The Bug Slayer']
+const COMMENT_TASK_LIST = `## Task List
+- [ ] Initial assessment - Please record your decision in the comment below
+  - [ ] CodeQL
+  - [ ] Security Lab
+- [ ] CodeQL: Generate result set and post the URL in the comment
+- [ ] Security Lab assessment: 
+  - [ ] Assess the Vulnerability Impact, the Vulnerability Scope, and the False Positive ratio
```
Comment (anticomputer) [inline .github/actions/replicate/replicate.ts:14]:
```suggestion
  - [ ] Assess the Vulnerability Impact, the Vulnerability Scope, and the False Positive ratio based on the provided CodeQL result set
```

**Comment by anticomputer** [comment]
looks nice @xcorail :)

File: .github/actions/replicate/replicate.ts [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+| Code Maturity | | 
+| Documentation | | 
+
+- [ ] Reject
+- [ ] Reject with thank you reward
+- [ ] Reject with encouragement swag (Decision: Dev Advocacy)
+- [ ] Accept
+`
+
+const COMMENT_FIRST_SUBMISSION = `## :tada: Fist submission for this user :tada:`
```
Comment (yo-h) [inline .github/actions/replicate/replicate.ts:39]:
```suggestion
const COMMENT_FIRST_SUBMISSION = `## :tada: First submission for this user :tada:`
```

**Comment by xcorail** [comment]
Suggestions from @anticomputer and @yo-h applied
Thanks

**Pull request merged**